controllers/front/ProductController.php

```diff
@@ -544,7 +544,11 @@ public function displayAjaxRefresh()
                 false,
                 false,
                 true,
-                $this->isPreview() ? ['preview' => '1'] : []
+                $this->isPreview() ? [
+                    'preview' => '1',
+                    'id_employee' => Tools::getValue('id_employee'),
+                    'adtoken' => Tools::getValue('adtoken'),
```

Comment:
Will these variables be in the url all the time? Are there cases when there is only preview=1?

Reply:
If the product is active, we don't need these 3 variables on the url. But, for preview of products, we need them all.
Unfortunately, for the use case of combination product and when we switch the combination of the product, we don't send these variables that's cause a rejecting of the ajax query.